tests/wpt/webgl/tests/conformance/rendering/preservedrawingbuffer-leak.html

```diff
@@ -29,6 +29,7 @@
 <html>
 <head>
 <meta charset="utf-8">
+<meta name="timeout" content="long">
```

Comment:
Please add this change to https://github.com/servo/servo/blob/master/tests/wpt/webgl/tools/timeout.patch.

Reply:
Done.